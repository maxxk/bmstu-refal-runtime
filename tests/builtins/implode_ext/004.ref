$ENTRY Go
{
	= <Prout <Lenw<Implode_Ext 'Asdsd_23-sd' 1 '2345'>>>;
}