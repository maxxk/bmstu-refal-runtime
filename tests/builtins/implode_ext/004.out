1 Asdsd_23-sd 
